DeviceList.plist

```diff
@@ -186,398 +210,512 @@
 	</dict>
 	<key>iPhone10,5</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPhone 8 Plus</string>
 		<key>version</key>
 		<real>10.5</real>
 		<key>name</key>
 		<string>iPhone 8 Plus Global</string>
 	</dict>
 	<key>iPhone10,6</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPhone X</string>
 		<key>version</key>
 		<real>10.6</real>
 		<key>name</key>
 		<string>iPhone X Global</string>
 	</dict>
 	<key>iPod1,1</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPod Touch 1</string>
 		<key>version</key>
 		<real>1.1</real>
 		<key>name</key>
 		<string>iPod Touch (1 Gen)</string>
 	</dict>
 	<key>iPod2,1</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPod Touch 2</string>
 		<key>version</key>
 		<real>2.1</real>
 		<key>name</key>
 		<string>iPod Touch (2 Gen)</string>
 	</dict>
 	<key>iPod3,1</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPod Touch 3</string>
 		<key>version</key>
 		<real>3.1</real>
 		<key>name</key>
 		<string>iPod Touch (3 Gen)</string>
 	</dict>
 	<key>iPod4,1</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPod Touch 4</string>
 		<key>version</key>
 		<real>4.1</real>
 		<key>name</key>
 		<string>iPod Touch (4 Gen)</string>
 	</dict>
 	<key>iPod5,1</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPod Touch 5</string>
 		<key>version</key>
 		<real>5.1</real>
 		<key>name</key>
 		<string>iPod Touch (5 Gen)</string>
 	</dict>
 	<key>iPod7,1</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPod Touch 6</string>
 		<key>version</key>
 		<real>7.1</real>
 		<key>name</key>
 		<string>iPod Touch (6 Gen)</string>
 	</dict>
 	<key>iPad1,1</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPad</string>
 		<key>version</key>
 		<real>1.1</real>
 		<key>name</key>
 		<string>iPad (WiFi)</string>
 	</dict>
 	<key>iPad1,2</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPad</string>
 		<key>version</key>
 		<real>1.2</real>
 		<key>name</key>
 		<string>iPad 3G</string>
 	</dict>
 	<key>iPad2,1</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPad 2</string>
 		<key>version</key>
 		<real>2.1</real>
 		<key>name</key>
 		<string>iPad 2 (WiFi)</string>
 	</dict>
 	<key>iPad2,2</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPad 2</string>
 		<key>version</key>
 		<real>2.2</real>
 		<key>name</key>
 		<string>iPad 2 (GSM)</string>
 	</dict>
 	<key>iPad2,3</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPad 2</string>
 		<key>version</key>
 		<real>2.3</real>
 		<key>name</key>
 		<string>iPad 2 (CDMA)</string>
 	</dict>
 	<key>iPad2,4</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPad 2</string>
 		<key>version</key>
 		<real>2.4</real>
 		<key>name</key>
 		<string>iPad 2 (WiFi Rev. A)</string>
 	</dict>
 	<key>iPad2,5</key>
 	<dict>
+		<key>shortname</key>
+		<string>iPad mini</string>
```

Comment:
For consistency within the pod (iPod Touch, iPad Pro, Apple Watch, etc.), I think it makes sense to stylize this as `iPad Mini`. Or else we should switch to `iPod touch` as well, since Apple officially has touch lower-cased as well https://www.apple.com/ipod-touch/